cuplaConfig.cmake

```diff
@@ -24,6 +24,17 @@
 
 cmake_minimum_required(VERSION 3.11.0)
```

Comment:
note: one we move to 3.12.0+ we can remove the block below, since it will put the new policies automatically to `NEW`:

https://cmake.org/cmake/help/v3.12/manual/cmake-policies.7.html

> The cmake_minimum_required() command does more than report an error if a too-old version of CMake is used to build a project. It also sets all policies introduced in that CMake version or earlier to NEW behavior.